app/teachingCall/teachingCallStatus/templates/ModalAddInstructors.html

```diff
@@ -15,48 +15,24 @@ <h4 class="modal-title">Add Instructors</h4>
 						<h5 style="margin-top: 0px;">Instructors</h5>
 
 						<div class="btn-group instructor-toggle-btn-container">
-							<label  
-							ng-if="areAllSenateInvited() == false"
-							class="instructor-toggle-btn btn btn-sm"
-							id="courses-pivot"
-							ng-click="addSenateInstructors()">
-								Add Senate
-							</label>
-							<label  
-							ng-if="areAllSenateInvited()"
-							disabled class="instructor-toggle-btn btn btn-sm"
-							uib-tooltip="All Senate Instructors Added"
-							id="courses-pivot">
-								Add Senate
-							</label>
-							<label
-							ng-if="areAllFederationInvited() == false"
-							class="instructor-toggle-btn btn btn-sm"
-							id="courses-pivot"
-							ng-click="addFederationInstructors() == false">
-								Add Federation
-							</label>
-							<label
-							ng-if="areAllFederationInvited()"
-							disabled class="instructor-toggle-btn btn btn-sm"
-							uib-tooltip="All Federation Instructors Added"
-							id="courses-pivot">
-								Add Federation
-							</label>
-							<label
-							ng-if="areAllLecturersInvited() == false"
-							class="instructor-toggle-btn btn btn-sm"
-							id="courses-pivot"
-							ng-click="addLecturerInstructors() == false">
-								Add Lecturers
-							</label>
-							<label
-							ng-if="areAllLecturersInvited()"
-							disabled class="instructor-toggle-btn btn btn-sm"
-							uib-tooltip="All Lecturer Instructors Added"
-							id="courses-pivot">
-								Add Lecturers
-							</label>
+							<span ng-repeat="instructorTypeId in instructorTypes.ids">
+								<span ng-if="areThereInstructorsOfType(instructorTypeId)">
+									<label
+									ng-if="!areAllInstructorsOfTypeInvited(instructorTypeId)"
```

Comment:
Multi-line tags should not have attributes lined up with tag name. Fix this in the rest of the file as well.